String Theory/Readme.txt

```diff
@@ -0,0 +1,6 @@
+Description:
```

Comment:
Can you remove this line and replace it with Modules :

Reply:
What exactly are modules?
Like the stages of the project or something?